public/restaurant.js

```diff
@@ -0,0 +1,78 @@
+let path = window.location.pathname
+let pathArray = path.split('/')
+let id = pathArray.pop()
+//let restId = document.getElementById('id')
+let name = document.getElementById('name')
+let address = document.getElementById('address')
+let phone = document.getElementById('phone')
+let website = document.getElementById('webSite')
+let hours = document.getElementById('hours')
+let notes = document.getElementById('notes')
+let noteContainer = document.getElementById('noteList')
+
+async function getRest(id) {
+	let restItem = await fetch(`https://json-server.burlingtoncodeacademy.now.sh/restaurants/${id}`)
+		.then((response) => {
+			return response.json()
+		})
+		.then((dataJson) => {
+			return dataJson
+		})
+
+	restId.textContent = restItem.id
+	name.textContent = restItem.name
+	address.textContent = restItem.address
+	phone.textContent = restItem.phone
+	webSite.textContent = restItem.website
+	hours.textContent = restItem.hours
```

Comment:
Since some of these restaurants do not have hours in their json data, when setting this textContent, a blank bullet point shows up for the ones without an 'hours' property. You could add a check in here to either add default text to the list item or remove the list item from the html if there are no hours. Since you hard coded a list item for all properties, you would need to actually use a method (like removeChild) to remove the list item from the document, sort of like the [codepen lab](https://bootcamp.burlingtoncodeacademy.com/lessons/www/dom-scripting#lab_add_item_to_list) we did in class about adding and removing specific items from a list using DOM queries. If you were to dynamically add the list items using javascript, then you could write a check that only creates an element if there is a value for the hours property.